inject-groovy/src/main/groovy/io/micronaut/ast/groovy/InjectTransform.groovy

```diff
@@ -834,6 +844,9 @@ class InjectTransform implements ASTTransformation, CompilationUnitAware {
                         }
                     } else {
                         Object[] interceptorTypeReferences = methodAnnotationMetadata.getAnnotationNamesByStereotype(Around).toArray()
+                        if (hasConstraints) {
+                            interceptorTypeReferences = ArrayUtils.concat(interceptorTypeReferences, "io.micronaut.validation.Validated")
```

Comment:
Why is this necessary given https://github.com/micronaut-projects/micronaut-core/pull/1526/files#diff-08f2cbd92c48cc377b2cdd189badf286R817 ?